WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressMapPickerViewModel.swift

```diff
@@ -25,19 +25,20 @@ final class DefaultAddressMapLocalSearchProvider: AddressMapLocalSearchProviding
     }
 }
 
-@available(iOS 17, *)
 @Observable
 final class AddressMapPickerViewModel: NSObject {
     var searchQuery = "" {
         willSet {
             searchQueryContinuation.yield(newValue)
         }
     }
+    // Syncs with the focused state of the search text field in `AddressMapPickerView`.
+    var isSearchFocused: Bool = false
     private(set) var searchResults: [MKLocalSearchCompletion] = []
-    var region = MKCoordinateRegion(
+    var mapPosition: MapCameraPosition = .region(MKCoordinateRegion(
```

Comment:
I'm curious why are we using Cupertino coordinates here? Should we pass in some sort of Location service that uses `CLLocationManagerDelegate` for the merchant's location? (I see we already instantiate a `CLLocationManager` when configuring the map) Or not really needed?